File: 图片轮播器－oc/图片轮播器－oc/ZDFlowLayout.m
//
//  ZDFlowLayout.m
//  图片轮播器－oc
//
//  Created by zz on 16/2/27.
//  Copyright © 2016年 zz. All rights reserved.
//

#import "ZDFlowLayout.h"

@implementation ZDFlowLayout
-(void)prepareLayout{

    [super prepareLayout];
    
    self.itemSize = self.collectionView.bounds.size;
    self.minimumLineSpacing = 0;
    self.minimumInteritemSpacing = 0;
    self.scrollDirection = UICollectionViewScrollDirectionHorizontal;
    
    self.collectionView.bounces = NO;
    self.collectionView.showsHorizontalScrollIndicator = NO;
    self.collectionView.pagingEnabled = YES;
    
}
@end

File: 图片轮播器swift/图片轮播器swift/ZDFlowLayout.swift
//
//  ZDFlowLayout.swift
//  图片轮播器－swift
//
//  Created by zz on 16/3/1.
//  Copyright © 2016年 zz. All rights reserved.
//

import UIKit

class ZDFlowLayout: UICollectionViewFlowLayout {

    override func prepareLayout() {
        super.prepareLayout()
        
        itemSize = (collectionView?.bounds.size)!
        minimumInteritemSpacing = 0
        minimumLineSpacing = 0
        self.collectionView?.showsHorizontalScrollIndicator = false
        self.scrollDirection = .Horizontal
        self.collectionView?.bounces = false
        self.collectionView?.pagingEnabled = true
    }
}
